snippets/nordic-flpr/README.rst

```diff
@@ -0,0 +1,12 @@
+.. _nordic-flpr:
+
+Nordic FLPR snippet with execution from SRAM (nordic-flpr)
+##########################################################
```

Comment:
this is not really what snippets were intended for and I almost consider it bit of a snippets misuse.

A preferred solution would involve board variants, but I do understand the wish to not have to create variants of all existing `flpr` cpucluster variants + cpuapp variants for this to work properly.

Thus principle is approved for this PR, but should be an area to improve.

Improvement ideas to the hw model for this use-case are welcome, feel free to send direct message.